packages/sdk/.npmignore

```diff
@@ -1 +1,25 @@
-!**/dist
+
+
+integration/
+coverage/
+dist/
+keysN1/
+keysN2/
+config/
+
+# Ignore configs
+setupTests.ts 
+.eslintrc.cjs
+.prettierrc.json
+docker-compose.yml
+Dockerfile
+jest.config.cjs
+tsconfig.json
+tsconfig.*.json
+tsconfig.tsbuildinfo
+rollup.config.js
+**/__test__/*
+
+# Ignore random junk
+.DS_Store
+node_modules/**
```

Comment:
Unfortunately, this won't work as expected.

check this read [for `.npmignore`](https://docs.npmjs.com/cli/v9/using-npm/developers#testing-whether-your-npmignore-or-files-config-works)

<details>
  <summary>Log output</summary>
  
  > sygma-sdk/packages/sdk$ npm pack
npm notice 
npm notice 📦  @buildwithsygma/sygma-sdk-core@1.1.0
npm notice === Tarball Contents === 
npm notice 15.2kB CHANGELOG.md                                                    
npm notice 3.5kB  README.md                                                       
npm notice 8.9kB  dist-cjs/chains/EVM/EvmBridge/EvmBridge.js                      
npm notice 4.8kB  dist-cjs/chains/EVM/EvmBridge/EvmBridge.js.map                  
npm notice 449B   dist-cjs/chains/EVM/EvmBridge/index.js                          
npm notice 171B   dist-cjs/chains/EVM/EvmBridge/index.js.map                      
npm notice 1.5kB  dist-cjs/chains/EVM/index.js                                    
npm notice 194B   dist-cjs/chains/EVM/index.js.map                                
npm notice 3.7kB  dist-cjs/chains/EVM/utils.js                                    
npm notice 1.6kB  dist-cjs/chains/EVM/utils.js.map                                
npm notice 878B   dist-cjs/chains/index.js                                        
npm notice 152B   dist-cjs/chains/index.js.map                                    
npm notice 1.5kB  dist-cjs/chains/Substrate/index.js                              
npm notice 178B   dist-cjs/chains/Substrate/index.js.map                          
npm notice 110B   dist-cjs/chains/Substrate/types/index.js                        
npm notice 134B   dist-cjs/chains/Substrate/types/index.js.map                    
npm notice 5.4kB  dist-cjs/chains/Substrate/utils/depositFns.js                   
npm notice 3.6kB  dist-cjs/chains/Substrate/utils/depositFns.js.map               
npm notice 883B   dist-cjs/chains/Substrate/utils/getAssetBalance.js              
npm notice 428B   dist-cjs/chains/Substrate/utils/getAssetBalance.js.map          
npm notice 864B   dist-cjs/chains/Substrate/utils/getBasicFee.js                  
npm notice 407B   dist-cjs/chains/Substrate/utils/getBasicFee.js.map              
npm notice 707B   dist-cjs/chains/Substrate/utils/getNativeTokenBalance.js        
npm notice 437B   dist-cjs/chains/Substrate/utils/getNativeTokenBalance.js.map    
npm notice 2.4kB  dist-cjs/chains/Substrate/utils/index.js                        
npm notice 489B   dist-cjs/chains/Substrate/utils/index.js.map                    
npm notice 1.2kB  dist-cjs/chains/Substrate/utils/listenForEvent.js               
npm notice 783B   dist-cjs/chains/Substrate/utils/listenForEvent.js.map           
npm notice 2.2kB  dist-cjs/chains/Substrate/utils/loadAccounts.js                 
npm notice 1.2kB  dist-cjs/chains/Substrate/utils/loadAccounts.js.map             
npm notice 956B   dist-cjs/chains/Substrate/utils/retrieveChainInfo.js            
npm notice 738B   dist-cjs/chains/Substrate/utils/retrieveChainInfo.js.map        
npm notice 1.5kB  dist-cjs/chains/Substrate/utils/substrateSocketConnect.js       
npm notice 1.0kB  dist-cjs/chains/Substrate/utils/substrateSocketConnect.js.map   
npm notice 2.6kB  dist-cjs/connectors/Connectors.js                               
npm notice 1.3kB  dist-cjs/connectors/Connectors.js.map                           
npm notice 456B   dist-cjs/connectors/index.js                                    
npm notice 155B   dist-cjs/connectors/index.js.map                                
npm notice 5.5kB  dist-cjs/Contracts/Erc20DetailedFactory.js                      
npm notice 3.6kB  dist-cjs/Contracts/Erc20DetailedFactory.js.map                  
npm notice 1.7kB  dist-cjs/fee/basicFee.js                                        
npm notice 1.1kB  dist-cjs/fee/basicFee.js.map                                    
npm notice 4.7kB  dist-cjs/fee/feeOracle.js                                       
npm notice 2.6kB  dist-cjs/fee/feeOracle.js.map                                   
npm notice 522B   dist-cjs/fee/index.js                                           
npm notice 184B   dist-cjs/fee/index.js.map                                       
npm notice 1.1kB  dist-cjs/index.js                                               
npm notice 190B   dist-cjs/index.js.map                                           
npm notice 28.0kB dist-cjs/Sygma.js                                               
npm notice 17.5kB dist-cjs/Sygma.js.map                                           
npm notice 112B   dist-cjs/types/globals.js                                       
npm notice 115B   dist-cjs/types/globals.js.map                                   
npm notice 833B   dist-cjs/types/index.js                                         
npm notice 138B   dist-cjs/types/index.js.map                                     
npm notice 110B   dist-cjs/types/types.js                                         
npm notice 111B   dist-cjs/types/types.js.map                                     
npm notice 4.4kB  dist-cjs/utils/helpers.js                                       
npm notice 2.6kB  dist-cjs/utils/helpers.js.map                                   
npm notice 5.6kB  dist-cjs/utils/index.js                                         
npm notice 3.5kB  dist-cjs/utils/index.js.map                                     
npm notice 8.7kB  dist-esm/chains/EVM/EvmBridge/EvmBridge.js                      
npm notice 5.0kB  dist-esm/chains/EVM/EvmBridge/EvmBridge.js.map                  
npm notice 72B    dist-esm/chains/EVM/EvmBridge/index.js                          
npm notice 171B   dist-esm/chains/EVM/EvmBridge/index.js.map                      
npm notice 117B   dist-esm/chains/EVM/index.js                                    
npm notice 204B   dist-esm/chains/EVM/index.js.map                                
npm notice 3.0kB  dist-esm/chains/EVM/utils.js                                    
npm notice 1.6kB  dist-esm/chains/EVM/utils.js.map                                
npm notice 85B    dist-esm/chains/index.js                                        
npm notice 152B   dist-esm/chains/index.js.map                                    
npm notice 96B    dist-esm/chains/Substrate/index.js                              
npm notice 180B   dist-esm/chains/Substrate/index.js.map                          
npm notice 44B    dist-esm/chains/Substrate/types/index.js                        
npm notice 134B   dist-esm/chains/Substrate/types/index.js.map                    
npm notice 5.0kB  dist-esm/chains/Substrate/utils/depositFns.js                   
npm notice 3.6kB  dist-esm/chains/Substrate/utils/depositFns.js.map               
npm notice 736B   dist-esm/chains/Substrate/utils/getAssetBalance.js              
npm notice 413B   dist-esm/chains/Substrate/utils/getAssetBalance.js.map          
npm notice 729B   dist-esm/chains/Substrate/utils/getBasicFee.js                  
npm notice 393B   dist-esm/chains/Substrate/utils/getBasicFee.js.map              
npm notice 542B   dist-esm/chains/Substrate/utils/getNativeTokenBalance.js        
npm notice 420B   dist-esm/chains/Substrate/utils/getNativeTokenBalance.js.map    
npm notice 521B   dist-esm/chains/Substrate/utils/index.js                        
npm notice 514B   dist-esm/chains/Substrate/utils/index.js.map                    
npm notice 1.0kB  dist-esm/chains/Substrate/utils/listenForEvent.js               
npm notice 766B   dist-esm/chains/Substrate/utils/listenForEvent.js.map           
npm notice 2.0kB  dist-esm/chains/Substrate/utils/loadAccounts.js                 
npm notice 1.3kB  dist-esm/chains/Substrate/utils/loadAccounts.js.map             
npm notice 797B   dist-esm/chains/Substrate/utils/retrieveChainInfo.js            
npm notice 750B   dist-esm/chains/Substrate/utils/retrieveChainInfo.js.map        
npm notice 1.4kB  dist-esm/chains/Substrate/utils/substrateSocketConnect.js       
npm notice 1.1kB  dist-esm/chains/Substrate/utils/substrateSocketConnect.js.map   
npm notice 2.5kB  dist-esm/connectors/Connectors.js                               
npm notice 1.4kB  dist-esm/connectors/Connectors.js.map                           
npm notice 86B    dist-esm/connectors/index.js                                    
npm notice 165B   dist-esm/connectors/index.js.map                                
npm notice 5.3kB  dist-esm/Contracts/Erc20DetailedFactory.js                      
npm notice 3.7kB  dist-esm/Contracts/Erc20DetailedFactory.js.map                  
npm notice 1.5kB  dist-esm/fee/basicFee.js                                        
npm notice 1.2kB  dist-esm/fee/basicFee.js.map                                    
npm notice 4.1kB  dist-esm/fee/feeOracle.js                                       
npm notice 2.7kB  dist-esm/fee/feeOracle.js.map                                   
npm notice 129B   dist-esm/fee/index.js                                           
npm notice 193B   dist-esm/fee/index.js.map                                       
npm notice 142B   dist-esm/index.js                                               
npm notice 204B   dist-esm/index.js.map                                           
npm notice 27.9kB dist-esm/Sygma.js                                               
npm notice 17.8kB dist-esm/Sygma.js.map                                           
npm notice 46B    dist-esm/types/globals.js                                       
npm notice 115B   dist-esm/types/globals.js.map                                   
npm notice 58B    dist-esm/types/index.js                                         
npm notice 130B   dist-esm/types/index.js.map                                     
npm notice 44B    dist-esm/types/types.js                                         
npm notice 111B   dist-esm/types/types.js.map                                     
npm notice 3.6kB  dist-esm/utils/helpers.js                                       
npm notice 2.5kB  dist-esm/utils/helpers.js.map                                   
npm notice 4.6kB  dist-esm/utils/index.js                                         
npm notice 3.4kB  dist-esm/utils/index.js.map                                     
npm notice 2.7kB  package.json                                                    
npm notice 9.9kB  src/chains/EVM/EvmBridge/EvmBridge.ts                           
npm notice 39B    src/chains/EVM/EvmBridge/index.ts                               
npm notice 84B    src/chains/EVM/index.ts                                         
npm notice 3.3kB  src/chains/EVM/utils.ts                                         
npm notice 52B    src/chains/index.ts                                             
npm notice 181B   src/chains/Substrate/index.ts                                   
npm notice 434B   src/chains/Substrate/types/index.ts                             
npm notice 6.0kB  src/chains/Substrate/utils/depositFns.ts                        
npm notice 1.0kB  src/chains/Substrate/utils/getAssetBalance.ts                   
npm notice 861B   src/chains/Substrate/utils/getBasicFee.ts                       
npm notice 816B   src/chains/Substrate/utils/getNativeTokenBalance.ts             
npm notice 497B   src/chains/Substrate/utils/index.ts                             
npm notice 1.2kB  src/chains/Substrate/utils/listenForEvent.ts                    
npm notice 2.5kB  src/chains/Substrate/utils/loadAccounts.ts                      
npm notice 938B   src/chains/Substrate/utils/retrieveChainInfo.ts                 
npm notice 2.1kB  src/chains/Substrate/utils/substrateSocketConnect.ts            
npm notice 2.8kB  src/connectors/Connectors.ts                                    
npm notice 53B    src/connectors/index.ts                                         
npm notice 13.2kB src/Contracts/Erc20Detailed.d.ts                                
npm notice 4.2kB  src/Contracts/Erc20DetailedFactory.ts                           
npm notice 1.8kB  src/fee/basicFee.ts                                             
npm notice 4.6kB  src/fee/feeOracle.ts                                            
npm notice 96B    src/fee/index.ts                                                
npm notice 109B   src/index.ts                                                    
npm notice 30.9kB src/Sygma.ts                                                    
npm notice 75B    src/types/globals.ts                                            
npm notice 25B    src/types/index.ts                                              
npm notice 3.0kB  src/types/types.ts                                              
npm notice 3.8kB  src/utils/helpers.ts                                            
npm notice 5.0kB  src/utils/index.ts                                              
npm notice 5.5kB  types/src/chains/EVM/EvmBridge/EvmBridge.d.ts                   
npm notice 2.4kB  types/src/chains/EVM/EvmBridge/EvmBridge.d.ts.map               
npm notice 74B    types/src/chains/EVM/EvmBridge/index.d.ts                       
npm notice 176B   types/src/chains/EVM/EvmBridge/index.d.ts.map                   
npm notice 119B   types/src/chains/EVM/index.d.ts                                 
npm notice 209B   types/src/chains/EVM/index.d.ts.map                             
npm notice 2.4kB  types/src/chains/EVM/utils.d.ts                                 
npm notice 693B   types/src/chains/EVM/utils.d.ts.map                             
npm notice 87B    types/src/chains/index.d.ts                                     
npm notice 157B   types/src/chains/index.d.ts.map                                 
npm notice 209B   types/src/chains/Substrate/index.d.ts                           
npm notice 257B   types/src/chains/Substrate/index.d.ts.map                       
npm notice 563B   types/src/chains/Substrate/types/index.d.ts                     
npm notice 622B   types/src/chains/Substrate/types/index.d.ts.map                 
npm notice 3.2kB  types/src/chains/Substrate/utils/depositFns.d.ts                
npm notice 1.3kB  types/src/chains/Substrate/utils/depositFns.d.ts.map            
npm notice 864B   types/src/chains/Substrate/utils/getAssetBalance.d.ts           
npm notice 394B   types/src/chains/Substrate/utils/getAssetBalance.d.ts.map       
npm notice 735B   types/src/chains/Substrate/utils/getBasicFee.d.ts               
npm notice 354B   types/src/chains/Substrate/utils/getBasicFee.d.ts.map           
npm notice 641B   types/src/chains/Substrate/utils/getNativeTokenBalance.d.ts     
npm notice 398B   types/src/chains/Substrate/utils/getNativeTokenBalance.d.ts.map 
npm notice 523B   types/src/chains/Substrate/utils/index.d.ts                     
npm notice 519B   types/src/chains/Substrate/utils/index.d.ts.map                 
npm notice 692B   types/src/chains/Substrate/utils/listenForEvent.d.ts            
npm notice 376B   types/src/chains/Substrate/utils/listenForEvent.d.ts.map        
npm notice 1.1kB  types/src/chains/Substrate/utils/loadAccounts.d.ts              
npm notice 596B   types/src/chains/Substrate/utils/loadAccounts.d.ts.map          
npm notice 516B   types/src/chains/Substrate/utils/retrieveChainInfo.d.ts         
npm notice 326B   types/src/chains/Substrate/utils/retrieveChainInfo.d.ts.map     
npm notice 1.4kB  types/src/chains/Substrate/utils/substrateSocketConnect.d.ts    
npm notice 667B   types/src/chains/Substrate/utils/substrateSocketConnect.d.ts.map
npm notice 1.9kB  types/src/connectors/Connectors.d.ts                            
npm notice 821B   types/src/connectors/Connectors.d.ts.map                        
npm notice 88B    types/src/connectors/index.d.ts                                 
npm notice 170B   types/src/connectors/index.d.ts.map                             
npm notice 320B   types/src/Contracts/Erc20DetailedFactory.d.ts                   
npm notice 384B   types/src/Contracts/Erc20DetailedFactory.d.ts.map               
npm notice 756B   types/src/fee/basicFee.d.ts                                     
npm notice 378B   types/src/fee/basicFee.d.ts.map                                 
npm notice 1.7kB  types/src/fee/feeOracle.d.ts                                    
npm notice 598B   types/src/fee/feeOracle.d.ts.map                                
npm notice 131B   types/src/fee/index.d.ts                                        
npm notice 198B   types/src/fee/index.d.ts.map                                    
npm notice 144B   types/src/index.d.ts                                            
npm notice 209B   types/src/index.d.ts.map                                        
npm notice 13.8kB types/src/Sygma.d.ts                                            
npm notice 5.0kB  types/src/Sygma.d.ts.map                                        
npm notice 119B   types/src/types/globals.d.ts                                    
npm notice 214B   types/src/types/globals.d.ts.map                                
npm notice 60B    types/src/types/index.d.ts                                      
npm notice 135B   types/src/types/index.d.ts.map                                  
npm notice 3.0kB  types/src/types/types.d.ts                                      
npm notice 3.2kB  types/src/types/types.d.ts.map                                  
npm notice 2.3kB  types/src/utils/helpers.d.ts                                    
npm notice 705B   types/src/utils/helpers.d.ts.map                                
npm notice 2.8kB  types/src/utils/index.d.ts                                      
npm notice 1.1kB  types/src/utils/index.d.ts.map                                  
npm notice === Tarball Details === 
npm notice name:          @buildwithsygma/sygma-sdk-core          
npm notice version:       1.1.0                                   
npm notice filename:      @buildwithsygma/sygma-sdk-core-1.1.0.tgz
npm notice package size:  68.3 kB                                 
npm notice unpacked size: 452.7 kB                                
npm notice shasum:        b753d4b95ab8086928150a7f913f48abb971eb48
npm notice integrity:     sha512-TuJGncpHPG0K6[...]p2Md/lh2UgMkA==
npm notice total files:   207                                     
npm notice 
buildwithsygma-sygma-sdk-core-1.1.0.tgz

</details>

Reply:
Just add other folders not to ignore, all others will be ignored as stated in `.gitignore`

Reply:
@BeroBurny your log looks correct to me. e.g.: only ESM, commonjs, types, src folders and package.json and .md files are in package. 
What else do you think should be ignored?

Reply:
I've added `src`to `.npmignore`